sst/SimEngCoreWrapper.cc

```diff
@@ -212,9 +223,11 @@ std::vector<std::string> SimEngCoreWrapper::splitArgs(std::string strArgs) {
     }
   }
   if (escapeSingle || escapeDouble) {
-    std::cerr << "Parsing failed: Invalid format - Please make sure all "
-                 "characters/strings are escaped properly."
-              << std::endl;
+    std::cerr << R"(
```

Comment:
Whilst this is more verbose, it's still not telling the user which part of their input is failing i.e. what exactly has failed in its parsing

Reply:
Please check if the new format is sufficient.

Reply:
Much better!